core/src/main/scala/com/microsoft/azure/synapse/ml/recommendation/SAR.scala

```diff
@@ -257,49 +257,4 @@ trait SARParams extends Wrappable with RecommendationParams {
   setDefault(timeDecayCoeff -> 30, activityTimeFormat -> "yyyy/MM/dd'T'h:mm:ss", supportThreshold -> 4,
     ratingCol -> C.RatingCol, userCol -> C.UserCol, itemCol -> C.ItemCol, similarityFunction ->
       "jaccard", timeCol -> "time", startTimeFormat -> "EEE MMM dd HH:mm:ss Z yyyy")
-
```

Comment:
are these removed on purpose?

Reply:
Yes, I'm moving them to org.apache.spark.ml.recommendation.RecommendationHelper's RecommendationParams since there're other classes need these.